iac/mainnet-fork/scripts/run_nginx_anvil.sh

```diff
@@ -13,12 +13,16 @@ trap 'kill $(jobs -p)' SIGTERM
 HOST="0.0.0.0"
 PORT=8544
 ETHEREUM_HOST=$HOST:$PORT
+# Stripping double quotations from the mnemonic seed phrase
```

Comment:
This feels like the kind of bug that takes hours to figure out and seconds to fix

Reply:
precisely what happened 🙃